oracle/tests/test_jdbc.py

```diff
@@ -60,18 +60,17 @@ def test__get_connection_jdbc(instance, dd_run_check, aggregator, expected_tags,
     Test the _get_connection method using the JDBC client
     """
     check = Oracle(CHECK_NAME, {}, [instance])
-    check.use_oracle_client = False
-    con = mock.MagicMock()
+    check.use_jdbc = mock.Mock(return_value=True)
 
-    cx = mock.MagicMock(DatabaseError=RuntimeError)
-    cx.clientversion.side_effect = cx.DatabaseError()
+    con = mock.MagicMock()
+    pyoradb = mock.MagicMock()
 
     jdb = mock.MagicMock()
     jdb.connect.return_value = con
     jpype = mock.MagicMock(isJVMStarted=lambda: False)
 
     mocks = [
-        ('datadog_checks.oracle.oracle.cx_Oracle', cx),
+        ('datadog_checks.oracle.oracle.oracledb', pyoradb),
```

Comment:
do we need to mock this? it shouldn't get used, should it?

Reply:
Removed